mypy.ini

```diff
@@ -133,6 +133,8 @@ files = src/sentry/analytics/,
         src/sentry/utils/time_window.py,
         src/sentry/utils/yaml.py,
         src/sentry/web/decorators.py,
+        tests/sentry/eventstream/test_eventstream.py,
+        tests/sentry/eventstream/test_synchronized.py,
```

Comment:
can even do `tests/sentry/eventstream/` iirc

Reply:
i wish! the other tests in there don't pass yet